lib/utility/moduleHelper.ts

```diff
@@ -1,16 +1,23 @@
 import { execSync } from 'child_process';
 import { gte as versionIsGreaterThanOrEqualTo } from 'semver';
 import { warn } from './messageHelper';
+import { Environment } from '../Builder';
 
 type TInstalledModules = {
     [key: string]: string;
 };
 
+const isDevEnvironment = process.env.NODE_ENV === Environment.Dev;
```

Comment:
Dit is niet handig zo. Naast de duplicatie van de environment check die je doet (DRY), doe je ook de aanname dat de `NODE_ENV` environment variabele je environment gaat bevatten. Dit hou je liever buiten de library.

De environment is bekend in de builder en adapters, respectievelijk als `config` constructor param en `builderConfig` in de `apply` functie. Deze moet je dan mee gaan geven aan de `validateIfRequiredModuleIsInstalled` als variabele.